### ISSUE TITLE
Regression: Full output in test mode with state_verbose: False (Salt 2014.7.1)

### ISSUE THREAD

**Issue by HontoNoRoger** [saltstack/salt#20187]
In Salt 2014.7.1 there will be a complete output of every correct file if run in `test=True` mode with `state_verbose: False`. This wasn't the case in 2014.7.0 and should not be printed.

**Example output for test:**
```
root@1404-salt-master /srv # salt 'testMachine' state.highstate test=True
…
----------
          ID: /etc/zookeeper/conf/environment
    Function: file.managed
      Result: None
     Comment: The file /etc/zookeeper/conf/environment is in the correct state
     Started: 10:59:10.290356
    Duration: 2.847 ms
     Changes:   
----------
          ID: /etc/java-7-openjdk/security/java.security
    Function: file.replace
      Result: None
     Comment: No changes were made
     Started: 10:59:10.297413
    Duration: 1.646 ms
     Changes:   
----------
…
Summary
-------------
Succeeded: 67 (unchanged=67)
Failed:     0
-------------
Total states run:     67
```
**Correct output for normal mode:**
```
root@1404-salt-master /srv # salt 'testMachine' state.highstate
testMachine:

Summary
-----------
Succeeded: 0
Failed:   0
-----------
Total states run:    0
```

**Comment by djs52** [comment]
I think this is a duplicate of #19833 which has been fixed in the 2014.7 branch. We really need a 2014.7.2 ASAP!

**Comment by rallytime** [comment]
Thanks for the report. It looks like @djs52 is correct that this is fixed in the head of 2014.7 with #19988, which will be included in the 2014.7.2 release. 

@HontoNoRoger Are you in a position to give that a try and confirm that this has been fixed for you?

**Comment by HontoNoRoger** [comment]
Fix confirmed for head revision of 2014.7.

**Comment by HontoNoRoger** [comment]
Fix confirmed for release version 2014.7.2.
